lightbulb/commands/options.py

```diff
@@ -420,8 +454,15 @@ def number(
     Args:
         name (:obj:`str`): The name of the option.
         description (:obj:`str`): The description of the option.
+        localize (:obj:`bool`): Whether to localize this option's name and description. If :obj:`true`, then the
+            ``name`` and ``description`` arguments will instead be interpreted as localization keys from which the
+            actual name and description will be retrieved from. Defaults to :obj:`False`.
         default (:obj:`~hikari.undefined.UndefinedOr` [ ``D`` ]): The default value for the option.
-        choices (:obj:`~hikari.undefined.UndefinedOr` [ ``ChoicesT`` ]): TODO
+        choices (:obj:`~hikari.undefined.UndefinedOr` [ ``ChoicesT`` ]): The choices for the option. Any of the
+            following can be interpreted as a choice: a sequence of :obj:`~hikari.commands.CommandChoice`, a mapping
+            of choice name to choice value, a sequence of 2-tuples where the first element is the name
+            and the second element is the value, or a sequence of :obj:`float` where the choice name and value will
+            be the same.
```

Comment:
Is this docstring not just verbosely explaining what ``ChoicesT`` is? Can we really not let the user figure this out?
Perhaps for this docstring we can 
```suggestion
        choices (:obj:`~hikari.undefined.UndefinedOr` [ ``ChoicesT`` ]): The choices for the option.
```
and then further explain how to set choices out in a guide on the docs?

Reply:
I think most people probably wouldn't care to look for a guide for this and would instead benefit from a more detailed argument description in the docstring. It could also be mentioned in a guide.